src/test/modelinstance_test.cpp

```diff
@@ -517,6 +517,40 @@ TEST_F(TestLoadModel, CheckONNXModelHandling) {
     EXPECT_EQ(model_files.front(), directoryPath + "/test_onnx/1/my-model.onnx");
 }
 
+TEST_F(TestLoadModel, CheckTFLiteModelHandling) {
+    GTEST_SKIP() << "Skipping invalid TFLite format test";
```

Comment:
Why we skip the test? We don't have lightweight tf-lite format model?

Reply:
test is skipped because of OV bug in negative scenario which is targeted for 2023.1